packages/tasks/src/model-data.ts

```diff
@@ -27,8 +12,9 @@ export interface ModelData {
 	id: string;
 	/**
 	 * Whether or not to enable inference widget for this model
+	 * TODO(type it)
 	 */
-	inference: InferenceDisplayability;
+	inference: string;
```

Comment:
```suggestion
	inference: string | "Yes" | ...;
```

To keep some documentation while being lax?

Reply:
but see https://github.com/huggingface-internal/moon-landing/pull/10618 (internal) it's gonna change quite a bit